### PR TITLE
Fix bug in gefs.R (issue #119) and use ncdf4 instead of ncdf 

### PR DESCRIPTION
**Pull request by potterzot**
* fix bug in gefs.R (perhaps related to check4pkg)

* convert to ncdf4 instead of ncdf

* remove dependency on tidyr::gather (this can be removed from NAMESPACE if no other functions require it)

* convert '=' to '<-'

* add check4pkg function to gefs.R since it seems to need it to work correctly

### CODE REVIEW

File: R/gefs.R [hunk 1]
```diff
@@ -218,3 +217,14 @@ gefs_dimension_values <- function(dim, con = NULL, ...) {
   if (is.null(con)) con = gefs_CONNECT(...)
   con$dim[[dim]]$vals
 }
+
+#' Check that package is installed
+#' 
+#' @param x the package name
+check4pkg <- function(x) {
```
Comment (sckott) [inline R/gefs.R]: can you remove this fxn?  it's defined in the `zzz.R` file, for utility fxns https://github.com/ropensci/rnoaa/blob/5f5c89ac1376faa745d7a60064ebb11b8f0d0b28/R/zzz.r#L153-L160

File: man/check4pkg.Rd [hunk 2]
```diff
@@ -0,0 +1,15 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (sckott) [inline man/check4pkg.Rd]: And remove this man file

**Comment by sckott** [comment]
Don't worry about builds for now, they won't succeed b/c of problems with `rgdal` installation

**Comment by potterzot** [comment]
@sckott should be all set, thanks.

**Pull request merged**